software/Makefile

```diff
@@ -125,8 +130,10 @@ build_libs: build_tools
 	$(MAKE)  -C $(SDK_DIR) -j1 bootrom
 
 # run on x86
```

Comment:
This is just a nice to have, so users can individually generate specific NBFs rather than running all at once